lib/src/cli.dart

```diff
@@ -0,0 +1,119 @@
+
+library grinder.src.cli;
+
+import 'dart:async';
+import 'dart:convert' show JSON, UTF8;
+import 'dart:io';
+
+import 'package:args/args.dart';
+
+import 'singleton.dart';
+import 'utils.dart';
+import '../grinder.dart';
+
+// This version must be updated in tandem with the pubspec version.
+const String APP_VERSION = '0.6.6+3';
```

Comment:
Any reason to move this constant out of `lib/grinder.dart`? (if we're moving this, we should update the comment in the pubspec file).